src/resources/authoring/authoring.schema.json

```diff
@@ -336,7 +387,7 @@
     "template": {
       "description": "The template identifies the React component that implements the challenge",
       "type": "string",
-      "enum": ["ClutchGame", "EggGame", "EggSortGame", "FVEggGame", "FVEggSortGame", "GenomeChallenge", "GenomePlayground", "ZoomChallenge"]
+      "enum": ["ClutchGame", "EggGame", "EggSortGame", "FVEggGame", "GenomeChallenge", "GenomePlayground", "ZoomChallenge"]
```

Comment:
Not sure why the FVEggSortGame was removed from the list? Schema still approves of the authoring, despite the eggDrop-noseSpike definition showing FVEggSortGame as its template.

Reply:
This list is for "generic" templates that can have arbitrary properties. The templates for which the schema had been extended (FVGenomeChallenge and FVEggSortGame so far) are matched in the template-specific part of the schema.